app/styles/pages/bulletin-index.scss

```diff
@@ -24,15 +31,12 @@
     }
 
     footer {
-      bottom: $padding-m;
-      color: $white;
       font-size: 11px;
-      position: absolute;
-      width: 100%;
+      margin-top: 80px;
+    }
 
-      a {
-        color: $white;
-      }
+    footer, footer a {
```

Comment:
Selector should have depth of applicability no greater than 3, but was 4<br>Each selector in a comma sequence should be on its own line